contrib/grimshot

```diff
@@ -26,13 +26,24 @@ else
   NOTIFY=no
 fi
 
+if [ "$1" = "--cursor" ]; then
+  CURSOR="-c"
+  shift 1
+fi
+
+
+if [ "$1" = "--notify" ] && [ "$NOTIFY" = "no" ]; then
+  NOTIFY=yes
+  shift 1
+fi
```

Comment:
In order to avoid repeating this block (that's already checked above), I'd recommend doing something like:

```bash
POSITIONAL=()
NOTIFY=no
CURSOR=no

while [[ $# -gt 0 ]]; do
  key="$1"

  case $key in
    -n|--notify)
      NOTIFY=yes
      shift # past argument
      ;;
    -c|--cursor)
      CURSOR=yes
      shift # past argument
      ;;
    *)    # unknown option
      POSITIONAL+=("$1") # save it in an array for later
      shift # past argument
      ;;
  esac
done
```

Inspired by https://stackoverflow.com/a/14203146

I also think that we could make `--copy` and `--save` into flags following the same pattern -- I recall another PR or issue wanting to allow a "copy + save" action, and this would be a good approach/

Reply:
Sorry for late reply

> I also think that we could make `--copy` and `--save` into flags following the same pattern -- I recall another PR or issue wanting to allow a "copy + save" action, and this would be a good approach/

Would it make sense to do that in this PR or I should make a new one?

Reply:
I think it's best to address that in a separate PR to keep thing in scope here.